assets/src/block-templates/deep-dive/template.js

```diff
@@ -3,7 +3,7 @@ import mainThemeUrl from '../main-theme-url';
 const {__} = wp.i18n;
 
 const topic = ['core/column', {}, [
-  ['core/group', {className: 'group-stretched-link'}, [
```

Comment:
This isn't what I meant, I think that we can fully remove the templates from the plugin since they have been moved to the theme. 
If we don't want to remove them just yet, this class should stay.

Reply:
Change reverted!

Reply:
@mardelnet you don't think we can remove them from the plugin even though they are in the theme already?

Reply:
Just to be clear, are you suggesting completely removing from the plugin the templates Quick Links and Deep Dive?

Reply:
Yes, exactly 🙂 But it can also be done in a separate PR since it's not really related to the issue, as you prefer!

Reply:
The templates were removed, @mleray